Documentation/building/freebsd-instructions.md

```diff
@@ -0,0 +1,278 @@
+# FreeBSD build instructions
+
+## Disclaimer
+Instructions bellow may or may not give you what you want.
```

Comment:
typo

Reply:
In addition to the "bellow" typo, this is also a strange disclaimer.  Why have instructions if they "may or may not give you what you want?"  We should be clear about why they may not provide the desired results.